mlrun/api/main.py

```diff
@@ -347,6 +356,39 @@ def _cleanup_runtimes():
         close_session(db_session)
 
 
+def _push_run_notifications(db: mlrun.api.db.base.DBInterface, db_session):
+
```

Comment:
although its a private method, I think its worth adding docstring for it explaining the use etc..